### DISCUSSION TITLE
Is all code here intended to be compliable by the community?

### DISCUSSION THREAD

**Discussion by mrlacey** [microsoft/WindowsAppSDK#556] [Q&A]
[The VISX project](https://github.com/microsoft/ProjectReunion/tree/main/dev/VSIX) has a reference to a private, access controlled, internal [package source](https://github.com/microsoft/ProjectReunion/blob/main/dev/VSIX/Directory.Build.props#L7).
Because of this, it's not possible for anyone without access (i.e. the wider open-source developer community) to be able to compile the code. This has the potential to severely limit the ability for the community to contribute to this open-source project.

Will this always be the case?
If this only applies to some projects within this repository, can a list of those which do welcome contributions be clearly included in the code contribution/developer guidelines that are apparently coming? 

From: https://github.com/microsoft/ProjectReunion/blob/main/docs/contributor-guide.md
> Over time, we will add Project Reunion code and samples into this repo and provide code contribution/developer guidelines.

**Comment by wjk** [comment]
I believe that package source is only used to populate [these PackageReferences](https://github.com/microsoft/ProjectReunion/blob/dd87c67d737d529ce418735d3808e811f1cbc733/dev/VSIX/Extension/ProjectReunion.Extension.csproj#L76-L87). From what I can see, the VSIX subfolder contains only template data, not any actual code, and if you want to contribute to the templates, I would try removing that package source from the props file, removing the PackageReference tags from the project, and rebuilding. (You may also need to change the Identity element in the vsixmanifest file to instruct VS to treat your modified build as a different project than the unmodified upstream.) All of the other PackageReferences are visible on nuget.org. Hope this helps!
